### PR TITLE
Make the capybara save path configurable

### PR DESCRIPTION
**Pull request by cbaines**

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -113,7 +113,7 @@
 Capybara.configure do |config|
   config.run_server = false
   config.default_driver = :poltergeist
-  config.save_path = __dir__ + "/../tmp"
+  config.save_path = ENV["CAPYBARA_SAVE_PATH"] or (__dir__ + "/../tmp")
```
Comment (binaryberry) [inline spec/spec_helper.rb]: Found the official guidance if it's helpful :-) https://github.com/alphagov/styleguides/commit/8d732f5bc110dfc2dd822e3899775b42019c0227

Comment (rubenarakelyan) [inline spec/spec_helper.rb]: Ah OK, looks like the rule has been relaxed a bit.

Comment (cbaines) [inline spec/spec_helper.rb]: @rubenarakelyan I'm still happy to change it if you want? I put the bracketing in from the start in an attempt to make it clear.

Comment (rubenarakelyan) [inline spec/spec_helper.rb]: Given the current rules I'm happy since it's a pretty simple statement :)

**Comment by cbaines** [comment]
Bump on this, anyone willing to approve?

**Pull request merged**